src/app/view/components/dot-persona-selector/dot-persona-selector.component.scss

```diff
@@ -0,0 +1,70 @@
+@import "variables";
+@import "mixins";
+
+.dot-persona-selector {
+    .ui-overlaypanel-content {
+        padding: $basic-padding * 2 0;
+    
+        .searchable-dropdown__search {
+            margin: 0 $basic-padding * 2;
+    
+            .searchable-dropdown__search-icon {
+                right: $basic-padding;
+            }
+    
+            .searchable-dropdown__search-inputfield {
+                @include form-field-border;
+                padding: 0 $basic-padding * 4 0 $basic-padding * 2;
+            }
+        }
+    }
+}
+
+.dot-persona-selector__container {
+    align-items: center;
+    cursor: pointer;
+    display: flex;
+    height: 48px;
+    position: relative;
+    width: 206px;
+
+    dot-avatar {
+        margin-right: $basic-padding;
+    }
+
+    .dot-persona-selector__label-container{
+        overflow: hidden;
```

Comment:
These 3 properties:

```
        overflow: hidden;
        text-overflow: ellipsis;
        white-space: nowrap;
```

We are using it in a lot of places in this code, why no put this into a mixing that we can reuse?